application/gqlgen/post.go

```diff
@@ -159,11 +159,13 @@ func (r *postResolver) URL(ctx context.Context, obj *models.Post) (*string, erro
 // Kilograms resolves the `kilograms` property of the post query, converting float64 to string
```

Comment:
```suggestion
// Kilograms resolves the `kilograms` property of the post query as a pointer to a float64
```

Reply:
Thanks. I found that the first time through, but missed it on this branch.